### ISSUE TITLE
NSwag xml summary with base classes

### ISSUE THREAD

**Issue by OFark** [dj-nitehawk/FastEndpoints#153]
XML <summary> documentation isn't read from base class properties. 

I'm using a few endpoints that all share the same parameter, and so I put that in a base class. But the XML <summary> markup next to the properties doesn't carry through to the summary in the swagger documentation.

**Comment by wjax** [comment]
Have you tried specifying `<inheritdoc/>` ?

**Comment by OFark** [comment]
No, I have not I assume you mean at the top of the inherit class?

**Comment by dj-nitehawk** [comment]
the following works even without inheritdoc:
```csharp
/// <summary>
/// this is the base class
/// </summary>
public class RequestBase
{
    /// <summary>
    /// base class property
    /// </summary>
    public int MyBaseProperty { get; set; }
}

public class Request : RequestBase
{
    /// <summary>
    /// sub class id property
    /// </summary>
    public Guid Id { get; set; }
}
```
![screen-record-1655482036](https://user-images.githubusercontent.com/7043768/174335735-bcd660a7-d0d5-479e-a476-5c5802c10e76.png)

@OFark pls try again with v4.3 which was released a few hours ago.
